src/Workspaces/CSharp/Portable/Formatting/Rules/WrappingFormattingRule.cs

```diff
@@ -78,12 +78,6 @@ private void AddSpecificNodesSuppressOperations(List<SuppressOperation> list, Sy
             {
                 AddSuppressWrappingIfOnSingleLineOperation(list, tokens.Item1, tokens.Item2);
             }
-
-            var ifStatementNode = node as IfStatementSyntax;
-            if (ifStatementNode?.Else != null)
-            {
-                AddSuppressWrappingIfOnSingleLineOperation(list, ifStatementNode.Else.ElseKeyword, ifStatementNode.Else.Statement.GetFirstToken(includeZeroWidth: true));
```

Comment:
Yes, this is being handled elsewhere. Since we have a lack of test around this scenario, I am adding some tests